.travis.yml

```diff
@@ -16,7 +16,7 @@ services:
 before_install:
   - npm install -g less
   - gem install puppet puppet-lint
-  - wget 'https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-0.90.10.tar.gz'
+  - wget 'https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-0.90.10.tar.gz' --no-check-certificate
```

Comment:
It does, but see https://github.com/mozilla/socorro/pull/2669 and https://bugzilla.mozilla.org/show_bug.cgi?id=1144158 for context. This line breaks our build and it's the only workaround we have for the moment.